### DISCUSSION TITLE
SQLCipher support

### DISCUSSION THREAD

**Discussion by albrrt** [directus/directus#11669] [Feature Requests]
We often use SQLCipher to encrypt SQLite databases in standalone desktop applications.

It would be great if Directus could support [SQLCipher](https://www.zetetic.net/sqlcipher/) as well?

**Comment by rijkvanzanten** [comment]
What would support for SQLCipher look like when it comes to Directus? Is it a matter of enabling the SQLite extension and that's it? I'm not familiar with the software 🙂 

see also https://github.com/directus/directus/discussions/7758

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
